acsoo/addons.py

```diff
@@ -85,3 +103,57 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+def do_makepot(config, database, odoo_bin, addons_dirs, odoo_config):
+    cmd = [
+        odoo_bin,
+        'shell',
+        '-d', database,
+        '--log-level=info',
+        '--no-xmlrpc',
+    ]
+    if odoo_config:
+        cmd.extend([
+            '-c', odoo_config
+        ])
+    script_dir = os.path.dirname(__file__)
+    file_path = os.path.join(script_dir, 'make_pot_script')
+    proc = subprocess.Popen(
+        cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT)
+    with open(file_path) as file:
+        cmd = file.read()
+    installable_addons = get_installable_addons(addons_dirs).items()
+    for addon, addon_dir in installable_addons:
+        kwargs = {
+            'module_name': addon,
+            'module_path': os.path.join(addon_dir[0], addon),
+        }
+        module_cmd = cmd % kwargs
+        proc.stdin.write(module_cmd)
+    proc.stdin.close()
+    checklog_config = config.get_default_map().get('checklog', {})
+    ignore = checklog_config.get('ignore', [])
+    echo = checklog_config.get('echo', None)
+    with tempinput(proc.stdout.read()) as tempfilename:
+        do_checklog(tempfilename, ignore, echo)
```

Comment:
`log-level=error` does not work? Do we have situations where a simple `odoo shell` outputs "normal" errors?